### ISSUE TITLE
Project doen't build

### ISSUE THREAD

**Issue by Zyr00** [rostam/GraphTea#58]
I just clone the repo and tried to build the project but to no effect 
#### Uname -r
```
[redacted@arch GraphTea]$ uname -a 
Linux arch 5.5.4-arch1-1 #1 SMP PREEMPT Sat, 15 Feb 2020 00:36:29 +0000 x86_64 GNU/Linux 
```
#### Output:
```
[redacted@arch GraphTea]$ ./make.sh 
Buildfile: /home/redacted/projects/GraphTea/build.xml 
clean: 
   [delete] Deleting directory /home/redacted/projects/GraphTea/binary 
   [delete] Deleting directory /home/redacted/projects/GraphTea/build 
BUILD SUCCESSFUL 
Total time: 0 seconds 
Buildfile: /home/redacted/projects/GraphTea/build.xml 
depends: 
    [mkdir] Created dir: /home/redacted/projects/GraphTea/build 
    [mkdir] Created dir: /home/redacted/projects/GraphTea/binary 
    [mkdir] Created dir: /home/redacted/projects/GraphTea/binary/lib 
    [mkdir] Created dir: /home/redacted/projects/GraphTea/binary/plugins 
    [mkdir] Created dir: /home/redacted/projects/GraphTea/binary/extensions 
compile: 
    [javac] /home/redacted/projects/GraphTea/build.xml:67: warning: 'includeantruntime' was not set, defaulting to build.sysclasspath=last; set to false for repeatable builds 
    [javac] Compiling 878 source files to /home/tunes/projects/GraphTea/build 
    [javac] /home/redacted/projects/GraphTea/src/graphtea/extensions/io/GraphSaveObject.java:7: error: package javax.xml.bind does not exist 
    [javac] import javax.xml.bind.DatatypeConverter; 
    [javac]                      ^ 
    [javac] /home/redacted/projects/GraphTea/src/graphtea/plugins/main/core/AlgorithmUtils.java:75: warning: [dep-ann] deprecated item is not annotated with @Deprecated 
    [javac]     ArrayList<ArrayList<Integer>> getAdjList(BaseGraph<VertexType, EdgeType> g) { 
    [javac]                                   ^ 
    [javac] /home/redacted/projects/GraphTea/src/graphtea/plugins/main/core/AlgorithmUtils.java:96: warning: [dep-ann] deprecated item is not annotated with @Deprecated 
    [javac]     int getDegree(BaseGraph<VertexType, EdgeType> bg, int node) { 
    [javac]         ^ 
    [javac] /home/redacted/projects/GraphTea/src/graphtea/plugins/main/core/AlgorithmUtils.java:116: warning: [dep-ann] deprecated item is not annotated with @Deprecated 
    [javac]     ArrayList<VertexType> getNeighbors(BaseGraph<VertexType, EdgeType> g, VertexType source) { 
    [javac]                           ^ 
    [javac] /home/redacted/projects/GraphTea/src/graphtea/plugins/main/core/AlgorithmUtils.java:135: warning: [dep-ann] deprecated item is not annotated with @Deprecated 
    [javac]     ArrayList<VertexType> getNeighbors2(BaseGraph<VertexType, EdgeType> g, VertexType source) { 
    [javac]                           ^ 
    [javac] /home/redacted/projects/GraphTea/src/graphtea/extensions/io/GraphSaveObject.java:79: error: cannot find symbol 
    [javac]         return DatatypeConverter.printBase64Binary(getBytesOfGraph(g)); 
    [javac]                ^ 
    [javac]   symbol:   variable DatatypeConverter 
    [javac]   location: class GraphSaveObject 
    [javac] /home/redacted/projects/GraphTea/src/graphtea/extensions/io/GraphSaveObject.java:83: error: cannot find symbol 
    [javac]         return getGraphFromBytes(DatatypeConverter.parseBase64Binary(s)); 
    [javac]                                  ^ 
    [javac]   symbol:   variable DatatypeConverter 
    [javac]   location: class GraphSaveObject 
    [javac] /home/redacted/projects/GraphTea/src/graphtea/extensions/io/GraphSaveObject.java:87: error: cannot find symbol 
    [javac]         return getGraphSaveOobjectfromBytes(DatatypeConverter.parseBase64Binary(s)); 
    [javac]                                             ^ 
    [javac]   symbol:   variable DatatypeConverter 
    [javac]   location: class GraphSaveObject 
    [javac] Note: Some input files use or override a deprecated API. 
    [javac] Note: Recompile with -Xlint:deprecation for details. 
    [javac] Note: Some input files use unchecked or unsafe operations. 
    [javac] Note: Recompile with -Xlint:unchecked for details. 
    [javac] 4 errors 
    [javac] 4 warnings 
BUILD FAILED 
/home/redacted/projects/GraphTea/build.xml:67: Compile failed; see the compiler error output for details. 
Total time: 3 seconds 
Error: Unable to access jarfile binary/graphtea-main.jar 

```

**Comment by azinazadi** [comment]
which java compiler version are you using?

**Comment by Zyr00** [comment]
[redacted@arch ~]$ javac --version 
javac 13.0.2

**Comment by azinazadi** [comment]
The project compiles with java 7. some packages where removed in later java
versions. possible solution:
https://stackoverflow.com/questions/52502189/java-11-package-javax-xml-bind-does-not-exist

if you fixed it we would be happy for a pull request...

**Comment by Zyr00** [comment]
Fixed with pull request #59

**Comment by rostam** [comment]
Since I merged request from @Zyr00 and the compilation works for now. So, I will close this issue.
